packages/core/src/provider/base.ts

```diff
@@ -104,6 +105,9 @@ export class SmartAccountProvider<
       minPriorityFeePerBidDefaults.get(chain.id) ??
       100_000_000n;
 
+    this.maxPriorityFeePerGasEstimateBuffer =
+      opts?.maxPriorityFeePerGasEstimateBuffer ?? 33;
```

Comment:
does 33% come from somewhere in particular?

Reply:
what was being used right now